jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.java

```diff
@@ -84,16 +238,40 @@ public void addAction(final CaseRolesPresenter.CaseRoleAction action) {
         a.setOnclick(e -> action.execute());
 
         final HTMLElement li = getDocument().createElement("li");
+        if (!action.isEnabled()) {
+            addCSSClass(li, "disabled");
```

Comment:
Thinking from user experience, maybe it makes more sense for the actions dropdown to stay open when a user clicks on disabled _Remove all assignments_ i.e. that nothing changes in the UI after clicking in disabled area.
It could happen that a user clicks that area by accident whereas they wanted to click on _Assign_ action. Then the user needs to open dropdown again just to get to the action they really wanted.